$NetBSD: patch-package.json,v 1.2 2015/08/19 21:02:45 wiedi Exp $
We have to install the devDependencies to build, but some of them require binary targets, which aren't available for NetBSD

--- package.json.orig	2015-06-29 16:45:18.000000000 +0000
+++ package.json
@@ -77,14 +77,12 @@
     "grunt-contrib-watch": "^0.5.3",
     "grunt-esvm": "^1.0.1",
     "grunt-jscs": "^1.8.0",
-    "grunt-mocha": "^0.4.10",
     "grunt-replace": "^0.7.9",
     "grunt-run": "^0.2.3",
     "grunt-s3": "^0.2.0-alpha.3",
     "grunt-simple-mocha": "^0.4.0",
     "html-entities": "^1.1.1",
     "http-proxy": "^1.8.1",
-    "husky": "^0.6.0",
     "istanbul": "^0.2.4",
     "license-checker": "3.0.3",
     "load-grunt-config": "^0.7.0",
